tests/manager/openAccount/managerCanOpenAccount.spec.js

```diff
@@ -28,4 +51,59 @@ Test:
 Tips:
  1. Do not rely on the customer row id for the step 13. Use the ".last()" locator to get the last row.
 */
-});
\ No newline at end of file
+const bankManagerMainPage = new BankManagerMainPage(page); 
+const openAccountPage = new OpenAccountPage(page); 
+const customersListPage= new CustomersListPage (page);
+
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickOpenAccountButton();
+await openAccountPage.waitForOpened();
+await openAccountPage.selectCustomer(customerName);
+await openAccountPage.selectCurrency('Dollar');
+await openAccountPage.clickProcessButton();
+await openAccountPage.reload();
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickCustomersButton();
+await customersListPage.waitForOpened();
+await customersListPage.assertLastRowAccountNumberContainsText('');
```

Comment:
The assertion `assertLastRowAccountNumberContainsText('')` is checking for an empty account number. It should verify that the account number is not empty, possibly by checking that it contains some text or is not an empty string.